bom/compile/pom.xml

```diff
@@ -207,7 +207,7 @@
     <dependency>
       <groupId>org.eclipse.paho</groupId>
       <artifactId>org.eclipse.paho.client.mqttv3</artifactId>
-      <version>1.2.0</version>
+      <version>1.2.1</version>
```

Comment:
Why do you change the minimum version we are able to work with?
Is there any point why products must update their paho version if they want to bump the mqtt transport?
If semver is used, we can keep the `[1.2.0,2)` and there is no need to force `[1.2.1,2)` for our imported paho packages.